data/geany.glade

```diff
@@ -3427,6 +3427,23 @@
                                   </packing>
                                 </child>
                                 <child>
+                                  <object class="GtkCheckButton" id="check_always_auto_close_brackets">
+                                    <property name="label" translatable="yes">Auto-close brackets if matching bracket found</property>
```

Comment:
It probably would be better for it to read more like "Auto-closing bracket even if a matching bracket is found" or something.